src/sys/signal.rs

```diff
@@ -680,6 +680,22 @@ pub fn kill<T: Into<Option<Signal>>>(pid: ::unistd::Pid, signal: T) -> Result<()
     Errno::result(res).map(drop)
 }
 
+/// Send a signal to a process group [(see
+/// killpg(3))](http://pubs.opengroup.org/onlinepubs/9699919799/functions/killpg.html).
+///
+/// If `pgrp` less then or equal 1, the behavior is undefined.
```

Comment:
In compiler-speak, "Undefined behavior" means something specific and very bad.  It would be better to say "platform-specific".

Reply:
@asomers OK. I have updated it.